book/src/SUMMARY.md

```diff
@@ -27,6 +27,7 @@ SPDX-License-Identifier: MIT OR Apache-2.0
   - [Shared types](./bridge/shared_types.md)
   - [Attributes](./bridge/attributes.md)
   - [Traits](./bridge/traits.md)
+- [Common Issues](./common-issues.md)
 - [For Contributors: CXX-Qt Internals](./internals/index.md)
   - [Build System](./internals/build-system.md)
```

Comment:
does anything in the build system notes need updating ?

Reply:
The existing ones already fit, but I've now moved them around a bit to fit better with our split of `problem space`/`solution space`.